apps/judicial-system/web/src/components/Table/CaseFileTable/CaseFileTable.tsx

```diff
@@ -91,8 +92,12 @@ const CaseFileTable: FC<Props> = ({
                 </Text>
                 <Text variant="small">
                   {formatMessage(strings.submittedBy, {
-                    category: file.category,
-                    initials: getInitials(file.submittedBy),
+                    title: getRoleTitleFromCaseFileCategory(
+                      file.category || undefined,
```

Comment:
Please consider supporting `null` values in the method `getRoleTitleFromCaseFileCategory`. If not, then please consider using `??` instead of `||` as it is more appropriate when handling defaults for`undefined` or `null`.